packages/react-query/src/shared/index.ts

```diff
@@ -11,18 +11,4 @@ export * from './queryClient';
 export * from './types';
 export * from './hooks/types';
 
-export {
-  /**
-   * @deprecated this is an internal function
-   */
-  getClientArgs,
-} from '../internals/getClientArgs';
```

Comment:
cc @juliusmarminge we don't need this exported right?

Reply:
I dont think so?